site/content/docs/standard/content-styles/hover-effects/index.html

```diff
@@ -50,6 +50,32 @@
   {{< /twsnippet/demo >}}
 
 
+  <!-- Section: Note about usage with animation  -->
+  <section>
+    <h3>Showing the animation effect</h3>
+    <p class="mb-3">
+      As you can see in the above example, you can add any animation to any
+      element on the page by using the following classes:
+    </p>
+    <ul>
+      <li>
+        <code>transition</code> : Used to animate the chaged properties on
+        hover.
+      </li>
+
+      <li>
+        <code>duration-x</code> : Used to define the duration of the animation
+        where <code>x</code> is the time in melli seconds; ex:
```

Comment:
Please change `melli seconds` to `milliseconds`. Also, please add the link to Tailwind's duration docs to help our users to find the built-in classes. Here's the url: https://tailwindcss.com/docs/transition-duration.